docs/release-notes/version-3.1.md

```diff
@@ -5,6 +5,7 @@
 ### Bug Fixes
 
 * [#8561](https://github.com/netbox-community/netbox/issues/8561) - Include option to connect a rear port to a console port
+* [#8553](https://github.com/netbox-community/netbox/issues/8553) - Fix contacts and ASNs missing in the search dropdown and contacts not showing the number of assigned objects
```

Comment:
You can omit the changelog to avoid potential merge conflicts with other PRs.

Reply:
okay, I can do that. Do you add things to the changelog as an additional commit before merging or releasing or how does the "getting things into the changelog" process work?

Reply:
Whichever maintainer merges the PR is responsible for following up in the change log to ensure everything is recorded. (Admittedly it's not the best process but it works well enough with incurring any additional overhead.)